ketchup/watch_schedule_builder.py

```diff
@@ -24,6 +24,7 @@ def calculate_watch_schedule(episodes, end_date, start_date, per_day_limit=None)
         "is_possible": True,
         "realistic_end_date": None,
         "extra_episodes": None,
+        "per_day_limit": per_day_limit,
```

Comment:
Why did you add this to the response ? 

If it's necessary for the main responses, we can add it, but I don't see a reason for this. The program should already know what limit was passed into it, unless it's needed in a response later I'm not sure why we should add this

Reply:
@leepatrickworrall this still needs to be removed

Reply:
Without it, it provides KeyError results for all of the function tests that have been created. In the test, per_day_limit is being asserted in the response, checking to see if it is equal to per_day_limit, but I have since changed that to a value.

<img width="577" alt="Screenshot 2023-09-04 at 15 12 26" src="https://github.com/chellygel/AnimeKetchup/assets/121133431/2849ac4f-727e-4a28-9e4d-3b74c981f74b">

Is how the test function would previously look like.

<img width="1347" alt="Screenshot 2023-09-04 at 15 13 25" src="https://github.com/chellygel/AnimeKetchup/assets/121133431/f46b39b2-04e0-4057-8031-97a1b8338a33">

Are the errors that I would receive.

<img width="643" alt="Screenshot 2023-09-04 at 15 14 07" src="https://github.com/chellygel/AnimeKetchup/assets/121133431/bc7e3e32-d058-409f-bcea-8f9fa6e8bd09">

Now this is how per_day_limit is being asserted in the test.